neurovault/apps/statmaps/static/scripts/metadata-edit.js

```diff
@@ -0,0 +1,382 @@
+/*global window, document, jQuery, Handsontable, NVMetadata */
+(function ($) {
+  'use strict';
+
+  function isString(val) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['isString' is defined but never used.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/825092758/issues/source?bid=2767031&fileBranchId=2767194#l5)